android/src/main/java/com/amazonaws/ivs/reactnative/player/AmazonIvsView.kt

```diff
@@ -337,8 +343,10 @@ class AmazonIvsView(private val context: ThemedReactContext) : FrameLayout(conte
         data.putMap("playerData", playerData)
 
         reactContext.getJSModule(RCTEventEmitter::class.java).receiveEvent(id, Events.DATA.toString(), data)
-      }
-      else -> {}
```

Comment:
We should keep this `else -> {}` as removing it caused this issue https://github.com/aws/amazon-ivs-react-native-player/pull/113

Reply:
Yes that warning was actually the reason behind that specific change,  of adding a `else` statement I added the other possible states of the player inside the loop following android studio's/Kotlin's own suggestions, if you prefer I can revert this change.

Reply:
Ahh, great! just drop a comment here so future us does not yank these alternate cases and cause a regression 👍

Reply:
done!